### DISCUSSION TITLE
RASPIAUDIO Audio DAC doesn't work

### DISCUSSION THREAD

**Discussion by RafadlCampo** [probonopd/MiniDexed#234] [Q&A] [answered]
Hi,

I'm amazed and grateful for this great proyect. 

I need some help, I can't heard any sound form my RASPIAUDIO Audio DAC Hat Sound Card, I edited the minidexed.ini Sound Device, but still sound throw the raspberry jack
![20220514_231959](https://user-images.githubusercontent.com/105709655/168772082-12b24e71-db6a-4535-aca5-6852630e212b.jpg)
.

**Comment by luisfcorreia** [comment]
did you change minidexed.ini to make it output via i2s?

**Comment by sou7611** [comment]
I have the same exact board and it's working for me ... just uncommented the i2s line in minidexed.cfg

**Reply by probonopd** [reply]
...and comment or delete any other `SoundDevice=...` lines!

**Comment by RafadlCampo** [comment, marked as answer]
It works now properly!! Thank you for your help.

I edited the minidexed file in another computer, and it works.
